### PR TITLE
README rewrite

### PR DESCRIPTION
**Pull request by elifoster**
The README was a little eh, so I rewrote it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+Pretty much every API action in core MediaWiki is possible through a helper instance method. However, for things that are not supported, there is the `post` method, which submits a `POST` request (which works fine for things that require a `GET` request in the API), that takes a hash parameter to pass to the API. Through this helper method, any API can be accessed!
+
+## Limitations
+There is currently no extension support. Some APIs are also not supported because the majority of the testing occurs on Gamepedia, which often uses software that is not always up to date.
 
 ## Installation
 ### RubyGems
-```
-gem install mediawiki-butt
+``` shell
```
Comment (xbony2) [inline README.md]: no space ^

Comment (elifoster) [inline README.md]: Why? The Bundler instructions have a space between the ` and ruby.

Comment (xbony2) [inline README.md]: Yeah well... if Bunder was good at making markdown files, they'd follow [Github's examples better](https://help.github.com/articles/creating-and-highlighting-code-blocks/).

Comment (elifoster) [inline README.md]: No, I mean **our** Bundler installation instructions have the space. So should we use the GitHub example way to do it, or this? There isn't really a difference.

Comment (xbony2) [inline README.md]: Bony don't like spaces :o

**Comment by elifoster** [comment]
There you go @xbony2

**Comment by xbony2** [comment]
My ego can be happy.

**Pull request merged**